extensions/classifiers/SVMPredictionServiceSpec.js

```diff
@@ -41,38 +41,96 @@ describe('SVM prediction functions', function() {
       expect(kvalues).toEqual(expectedKvalues);
     });
 
-    it('should predict correct label', function() {
+    it('should give correct labels and confidence estimations', function() {
       var svmData = {
-        classes: [0, 1],
-        n_support: [2, 2],
-        support_vectors: [[1.0, 1.0], [0.0, 0.0], [1.0, 0.0], [0.0, 1.0]],
-        dual_coef: [[1.0, 1.0, -1.0, -1.0]],
-        intercept: [0.0],
-        probA: [],
-        probB: [],
+        classes: [0.0, 1.0],
```

Comment:
Why 0.0 and 1.0 rather than 0 and 1? Thought class labels should be integers.

Can you explain in a comment inline whether this example represents two non-default classes or just one?

Reply:
Yes, well, that's because my training data (I created a synthetic dataset to test this service which is not related to answer classification in our scenario)  had 1.0 and 0.0 as labels, so extracted classifier has the same. I will modify it.

Reply:
OK, thanks. Worth noting that last point in a code comment, I think.